README.md

```diff
@@ -38,6 +38,8 @@ conda install -c conda-forge jhub-apps
 conda env create -f environment-dev.yml
 conda activate jhub-apps-dev
 pip install -e .
+cd ui
+npm install
```

Comment:
I would argue that this should actually be in Step 2 in the Developing Locally section. You should not need NPM dependencies installed to just run the app with the prebuilt assets. You just need those when you want to modify the ui part of the app.

That said, it doesn't hurt to add here, but maybe also add to the Developing Locally section

Reply:
To me the Developing Locally section sounds like: "steps to run every time when developing locally after you already installed the dependencies" - as it does not install installation of other dependencies. That said, I see what you mean, what about:

~~~suggestion
```

To develop the React UI frontend, also run:

```bash
cd ui
npm install
cd -
```
~~~

Reply:
> To me the Developing Locally section sounds like: "steps to run every time when developing locally after you already installed the dependencies" - as it does not install installation of other dependencies. That said, I see what you mean, what about:

Works for me.